tests/network/vmi_multus.go

```diff
@@ -1550,6 +1553,16 @@ func getInterfaceNetworkNameByMAC(vmi *v1.VirtualMachineInstance, macAddress str
 	return ""
 }
 
+func getInterfaceInfoSourceByMAC(vmi *v1.VirtualMachineInstance, macAddress string) string {
```

Comment:
I think this should be a general lookup based on mac, then used as:
`lookupInterfaceStatusByMac(vmi.Status.Interfaces, mac).InfoSource`

I guess it fits `vmispec`

Reply:
done